kolibri/core/assets/src/api-resources/exam.js

```diff
@@ -7,6 +7,15 @@ class ExamResource extends Resource {
   static idKey() {
     return 'id';
   }
+
+  /**
+   * Returns a Collection of Exams filtered by a collectionId
+   * @param {string} collectionId - e.g. a classId
+   * @returns {Collection<Exam>}
+   */
+  getCollectionForClass(collectionId) {
```

Comment:
I wanted to add this to Exam Resource class to 'hide' the Python away from JS enduser (e.g. user doesn't need to think about how to query Exam.objects to get what they want), and put in some documentation for people to read.